CHANGELOG.md

```diff
@@ -4,94 +4,67 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-
-## [unReleased] - 2021-MM-DD
+## \[unReleased\] - 2021-MM-DD
 
 ### Added
 
 - Added `state_id` property to the `Callback` base class ([#6886](https://github.com/PyTorchLightning/pytorch-lightning/pull/6886))
 
-
--
-
-
 -
 
+*
 
 -
 
 ### Changed
 
 - Replace `iteration_count` and other index attributes in the loops with progress dataclasses ([#8477](https://github.com/PyTorchLightning/pytorch-lightning/pull/8477))
 
-
```

Comment:
Is it okay that two empty lines between changelogs are now only one line?

Reply:
well we can ignore chnagelog...